### PR TITLE
[T4A3][F12-A2] Lee Jun Han Bryan

### PR DESCRIPTION
**Pull request by bryanleejh**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -3,57 +3,35 @@
...
+	/**
+	 * Validates given address.
+	 *
+	 * @throws IllegalValueException
+	 *             if given address string is invalid.
+	 */
+	public Address(String address, boolean isPrivate) throws IllegalValueException {
+		String trimmedAddress = address.trim();
+		this.isPrivate = isPrivate;
+		if (!isValidAddress(trimmedAddress)) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Address.java:24]: Is there a way to move validation into `Contact` class?
Recall from tutorial.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public abstract class Contact {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Contact.java:3]: Do remember to add header comments,

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+	public String toString() {
+		return value;
+	}
+
+	public int hashCode() {
+		return value.hashCode();
+	}
+
+	public boolean isPrivate() {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Contact.java:15]: Recall from our tutorial on commenting of methods.

**Review by brandonyeoxg — COMMENTED** [review]
@bryanleejh Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Address.java (hunk 1 above)
Comment (bryanleejh) [reply src/seedu/addressbook/data/person/Address.java:24]: Hmm can't remember - will ask you again!

**Comment by bryanleejh** [comment]
Thanks Brandon!

**Pull request closed without merging**